src/main/java/ca/uhn/fhir/jpa/starter/JpaRestfulServer.java

```diff
@@ -170,21 +172,42 @@ protected void initialize() throws ServletException {
             registerProvider(retriggeringProvider);
         }
 
+        // Define your CORS configuration. This is an example
+        // showing a typical setup. You should customize this
+        // to your specific needs
+        CorsConfiguration config = new CorsConfiguration();
```

Comment:
It'd probably make sense for this to be exposed in the HapiProperties file as a config item (maybe with the allowed origins being modifiable?)

Reply:
This would be great.

Reply:
Thank you, have updated it to be configurable based on your suggestion. Also removed the unused lib on pom.xml and cors configuration on web.xml.